front/src/common/ScenarioExplorator/ScenarioExplorator.tsx

```diff
@@ -88,7 +103,7 @@ export default function ScenarioExplorator() {
       role="button"
       tabIndex={0}
     >
-      {projectDetails && studyDetails && scenarioDetails ? (
+      {projectID && projectDetails && studyID && studyDetails && scenarioID && scenarioDetails ? (
```

Comment:
```suggestion
      {projectDetails && studyDetails && scenarioDetails ? (
```
I think the previous test is as efficient as this one but with less variables :)

Reply:
Yes, but ...Details are by managed by the state of the component and their lifecycle prooved less reliable. Componenent is not right controlled

Reply:
Ok ok, I agree !